docs/serving/using-auto-tls.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Enabling automatic TLS certificate provisioning"
+linkTitle: "Enabling auto TLS certs"
+weight: 64
+type: "docs"
+---
+
+If you install and configure cert-manager, you can configure Knative to
+automatically obtain new TLS certificates and renew existing ones.
+To learn more about using secure connections in Knative, see
+[Configuring HTTPS with TLS certificates](./using-tls-cert.md).
+
+## Before you begin
+
+You must meet the following prerequisites to enable automatic certificate
+provisioning:
+
+- The following must be installed on your Knative cluter:
+  - [Knative Serving version 0.6.0 or higher](../install/).
+  - [Istio with SDS, version 1.1 or higher](../install/installing-istio.md#installing-istio-with-SDS-to-secure-the-ingress-gateway).
+    Note: Currently, [Gloo](https://github.com/solo-io/gloo) is unsupported.
+  - [cert-manager version `0.6.1` or higher](./installing-cert-manager.md).
+- Your Knative cluster must be configured to use a
+  [custom domain](./using-a-custom-domain.md).
+
+
+## Enabling automatic certificate provisioning
+
+To enable Knative to automatically provision TLS certificates:
+
+1. Run the following commands to install the `networking-certmanager`:
+
+   ```shell
+   # KNATIVE_VERSION needs to be 0.6.0 or above.
+   KNATIVE_VERSION=0.6.0
+
+   kubectl apply --filename https://github.com/knative/serving/releases/download/v${KNATIVE_VERSION}/serving.yaml \
+   --selector networking.knative.dev/certificate-provider=cert-manager
+    ```
+
+1. Configure your CA, the required `DNS-01` challenge type, and your DNS
+   provider in the `config-certmanager` ConfigMap, where:
+
+    - `issuerRef`: Defines the CA that issues your TLS certificates.
+    - `solverConfig`: Defines the challenge type and DNS provider.
+
+   Example:
+
+   If you use the Let's Encrypt CA and Google Cloud DNS, then the `name` of
```

Comment:
Below `issuerRef` and `solverConfig` examples are specific for the [ClusterIssuer](https://github.com/knative/docs/blob/master/docs/serving/using-cert-manager-on-gcp.md#specifying-a-certificate-issuer) example. `letsencrypt-issuer` is the actually `name` of the `ClusterIssuer`, and the `provider` field in the `solverConfig` also references the `providers` in the `ClusterIssuer` example.
So I think we need to tell users that if they create `ClusterIssuer` like the example, then below are how the corresponding `issuerRef` and `solverConfig` look like.

Reply:
is this the right file we want configured (can i link it here)?
https://github.com/knative/serving/blob/master/config/config-certmanager.yaml

Reply:
Yes, you can link it here.

Reply:
@ZhiminXiang revised PTAL